agent/internal/backend/api/api.go

```diff
@@ -816,15 +816,15 @@ func NewBlockedUserEventPropertiesFromFace(that BlockedUserEventPropertiesFace)
 
 type BlockedUserEventPropertiesFace interface {
 	GetActionId() string
-	GetOutput() BlockedUserEventProperties_Output
+	GetOutput() BlockedUserEventPropertiesOutput
 }
 
-type BlockedUserEventProperties_OutputFace interface {
+type BlockedUserEventPropertiesOutputFace interface {
 	GetUser() map[string]string
 }
 
-func NewBlockedUserEventProperties_OutputFromFace(that BlockedUserEventProperties_OutputFace) *BlockedUserEventProperties_Output {
-	this := &BlockedUserEventProperties_Output{}
+func NewBlockedUserEventPropertiesOutputFromFace(that BlockedUserEventPropertiesOutputFace) *BlockedUserEventPropertiesOutput {
+	this := &BlockedUserEventPropertiesOutput{}
 	this.User = that.GetUser()
```

Comment:
I don't like the `this`/`that` naming convention. I'd rather have this shortened but meaningful, like `output.User = outputFace.GetUser()` or `output.User = face.GetUser()` or whatever, which is much more readable and meaningful. (This comment is valid for some other places)